applications/signals.py

```diff
@@ -0,0 +1,13 @@
+from django.db.models.signals import post_save
+from django.dispatch import receiver
+
+from applications import models
+
+
+# Delete DraftApplication when application submitted
+@receiver(post_save, sender=models.Application)
+def clean_draftapplication(sender, instance, created, *args, **kwargs):
```

Comment:
Can we rename this to `clean_draft_application`?
```suggestion
def clean_draft_application(sender, instance, created, *args, **kwargs):
```

Reply:
Feel free to check me on this, I know it's also a model name